app/controllers/api/owners_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::OwnersController < Api::BaseController
+  include PublishersHelper
+  include ActionController::Serialization
+
+  def index
+    render(json: Publisher.all)
```

Comment:
hmmm this is going to return Everything

we may have to add pagination if we use this more than once.